README.md

```diff
@@ -107,3 +128,69 @@ You should be aware that setting this variable provides a safe *subset* of
 functionality; it is still possible that PRs raised for code repositories will
 still fail under the automated CI systems since those systems are running all
 possible tests.
+
+
+## Run the Kata Containers tests
+
+### Requirements to run Kata Containers tests
+
+You need to install the following to run Kata Containers tests:
+
+- [golang](https://golang.org/dl)
+
+  To view the versions of go known to work, see the `golang` entry in the
+  [versions database](https://github.com/kata-containers/runtime/blob/master/versions.yaml).
+
+- `make`.
+
+### Prepare an environment
+
+The recommended method to set up Kata Containers is to use the official and latest
+stable release. You can find the official documentation to do this in the
+[Kata Containers installation user guides](https://github.com/kata-containers/documentation/blob/master/install/README.md).
+
+To try the latest commits of Kata use the CI scripts, which build and install from the
+kata-containers repositories, with the following steps:
+
+> **Warning:** This may replace/delete packages and configuration that you already have.
+> Please use these steps only on a testing environment.
+
+Add the `$GOPATH/bin` directory to the PATH:
+```
+$ export PATH=${GOPATH}/bin:${PATH}
+```
+
+Clone the `kata-container/tests` repository:
+```
+$ go get github.com/kata-containers/tests
+```
+
+Go to the tests repo directory:
+```
+$ cd $GOPATH/src/github.com/kata-containers/tests
+```
+
+$ Execute the setup script:
+```
+.ci/setup.sh
```

Comment:
I think the dollar got moved to line 173 by mistake? :)

Reply:
ouch, good catch, fixed :)